mock-interactions.js

```diff
@@ -240,7 +240,46 @@
     makeMouseEvent('mouseup', xy, node);
   }
 
-  /**
+    /**
+     * Fires a `mouseenter` mouse event on a specific node, at a given set of coordinates.
+     * This event bubbles and is cancellable. If the (x,y) coordinates are
```

Comment:
`mouseenter` doesn't bubble and is not cancellable.
Also, it should fire a separate event for each parent up to `<body>` - see https://developer.mozilla.org/en-US/docs/Web/Events/mouseenter. Do we expect this to be done here or by the user?